node/src/blockchain/blockchain_interface.rs

```diff
@@ -138,18 +139,17 @@ pub trait BlockchainInterface {
         recipient: &Wallet,
     ) -> Result<RetrievedBlockchainTransactions, BlockchainError>;
 
-    fn chain(&self) -> Chain;
+    fn mobilize_payable_payments_agent(&self) -> Box<dyn PayablePaymentsAgent>;
```

Comment:
"Mobilize" is not a word most developers will expect to see in a place like this. It's a unique word, and I like it, but the fact that it's unique means that readers will need to spend time figuring out what it means. (Argh! I like it!) So...if I were inventing this method, I'd probably use a word like "create" or "make" or "build" instead of "mobilize"; but I still like "mobilize."